client/my-sites/email/email-management/home/email-plan-subscription.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { CompactCard } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import RenewButton from 'calypso/my-sites/domains/domain-management/edit/card/renew-button';
+import AutoRenewToggle from 'calypso/me/purchases/manage-purchase/auto-renew-toggle';
+import { withLocalizedMoment } from 'calypso/components/localized-moment';
+
+class EmailPlanSubscription extends React.Component {
+	renderDefaultRenewButton() {
+		const { domain, purchase, selectedSite, isLoadingPurchase, translate } = this.props;
+
+		if ( ! domain.currentUserCanManage ) {
+			return null;
+		}
+
+		if ( ! purchase && ! isLoadingPurchase ) {
+			return null;
+		}
+
+		return (
+			<RenewButton
+				compact={ true }
```

Comment:
Note you don't need to specify `true` when a property is true:
```suggestion
				compact
```

Reply:
I'm aware but my personal preference is to skip this type of syntactic sugar... it just looks weird and if you ask me makes the code less readable (specifically when all the properties are listed on one line)